File: Pyramid Pattern/code9.c
#include <stdio.h>

void main (){

	int row;
	printf("Enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 0; i < row ; i++){

		for(int a = 0; a < i;a++ ){

			printf(" \t");
		}
		for(int j = 0; j < ((row-i)*2)-1;j++){

			printf("%d\t",j+1);

		}
		printf("\n\n");

	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/7/code9.java
import java.util.Scanner;

class NestedForDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("Enter no. of rows : ");
		int row = s.nextInt();
	

		for(int i = 1; i <= row ; i++){

			for(int j = 0; j < row ; j++){

				System.out.print((char)(65+j)+"	");

			}
			System.out.println("\n");

		}

	}


}
